src/_guides/libraries/library-tour.md

```diff
@@ -2369,289 +1646,65 @@ The dart:convert library also has converters for ASCII and ISO-8859-1
 (Latin1). For details, see the [API docs for the dart:convert library.][dart:convert]
 
 
-## dart:mirrors - reflection
```

Comment:
Why remove the section for `dart:mirrors`? Was some of the content outdated for 1.24?

Reply:
Because mirrors aren't supported on every platform. This tour concentrates on the libraries that people can rely on everywhere.

We could move the dart:mirrors content somewhere else. (In fact, I meant to but accidentally didn't. Then I thought maybe that was serendipitous... :)